### ISSUE TITLE
BLOB and unicode failing in executemany

### ISSUE THREAD

**Issue by elidaian** [PyMySQL/PyMySQL#380]
Hi,

I found a failure when using `cursor.executemany`, whereas using `cursor.execute` in a loop would not raise any exception.

For example, using this table:

```sql
CREATE TABLE users (
  id INTEGER NOT NULL AUTO_INCREMENT,
  username VARCHAR(16) NOT NULL,
  password BLOB(16),
  PRIMARY KEY (id),
  UNIQUE (username)
)
```

Define the following globals:

```python
import hashlib

def md5(x):
    return hashlib.md5(x).digest()

USERS = [
    (u'user1', md5('user1')),
    (u'user2', md5('user2'))
]
```

Now, create a `connection` object (with the `'utf8'` encoding, if it matters), and let's go.

Trying to run this:

```python
    with connection.cursor() as cursor:
        sql = 'INSERT INTO users (username, password) VALUES  (%s, %s)'
        cursor.executemany(sql, USERS)
```

would give us:

```python
Traceback (most recent call last):
  File "C:/Users/user/PycharmProjects/pymysqlbug/bugged.py", line 48, in <module>
    cursor.executemany(sql, USERS)
  File "C:\Python27\lib\site-packages\pymysql\cursors.py", line 155, in executemany
    self._get_db().encoding)
  File "C:\Python27\lib\site-packages\pymysql\cursors.py", line 169, in _do_execute_many
    v = values % escape(next(args), conn)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc9 in position 2: ordinal not in range(128)
```

However, this code runs just fine:
```python
    with connection.cursor() as cursor:
        sql = 'INSERT INTO users (username, password) VALUES  (%s, %s)'
        for user in USERS:
            cursor.execute(sql, user)
```

I have expected both codes to do the same.

Thanks for your help!

**Comment by methane** [comment]
I've released 0.6.7.
Is your issue reproducible on it?

**Comment by elidaian** [comment]
It worked for my in 0.6.7.
Thanks!
